src/tests/conformance/post_handshake/query/get_block.rs

```diff
@@ -0,0 +1,59 @@
+use tempfile::TempDir;
+
+use crate::{
+    setup::node::Node,
+    tools::{rpc, synthetic_node::SyntheticNodeBuilder},
+};
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c003_V1_BLOCK_ROUND_get_block() {
+    // ZG-CONFORMANCE-003
+
+    // Spin up a node instance.
+    let target = TempDir::new().expect("Couldn't create a temporary directory");
+    let mut node = Node::builder()
+        .build(target.path())
+        .expect("Unable to build the node");
+    node.start().await;
+
+    // Create a synthetic node and enable handshaking.
+    let synthetic_node = SyntheticNodeBuilder::default()
+        .build()
+        .await
+        .expect("Unable to build a synthetic node");
+
+    // Connect to the node and initiate the handshake.
+    synthetic_node
+        .connect(node.net_addr().unwrap())
+        .await
+        .unwrap();
```

Comment:
Sometimes there is `unwrap`, sometimes `expect`. I feel like `expect` could help if test fails.